typing/typecore.ml

```diff
@@ -107,11 +107,15 @@ let type_module =
   ref ((fun _env _md -> assert false) :
        Env.t -> Parsetree.module_expr -> Typedtree.module_expr)
 
+let gen_mod_ident : (Parsetree.module_expr -> Ident.t option) ref = ref (fun _ -> assert false)
```

Comment:
Could we have a comment on top of this forward declaration, with the declaration site, as for the others above and below?